### ISSUE TITLE
Can't login if user has only one IAM role

### ISSUE THREAD

**Issue by dfevre** [Versent/saml2aws#48]
Users can't login to AWS if they are configured in the IdP with a single IAM role only.

If a user is configured with only a single IAM role in the IdP, `saml2aws` used to log the user in without prompting them to select a role. If they have more than one role, they are presented with a list of roles they must select from. 

This was working in version 1.7.1 but with 1.8.1 this no longer works. If the user is configured with one role only, they are now prompted to select a role from the list but no list is printed. Entering `0` causes it to crash.

```
$ saml2aws login --provider=KeyCloak
Adding osx helper
Hostname [hostname.com.au/auth/realms/amp/protocol/saml/clients/amazon-aws]:
To use saved username and password just hit enter.
Username [user]:
Password: **********

Authenticating as user to KeyCloak https://hostname.com.au/auth/realms/amp/protocol/saml/clients/amazon-aws
Please choose the role you would like to assume:
Selection: 0
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/versent/saml2aws.PromptForAWSRoleSelection(0x477cdd8, 0x0, 0x0, 0x477cdd8, 0x0, 0x0)
	/Users/markw/Code/go/src/github.com/versent/saml2aws/input.go:88 +0x602
github.com/versent/saml2aws/cmd/saml2aws/commands.resolveRole(0xc42000e080, 0x1, 0x1, 0xc4204fc000, 0x29d4, 0xc4201461c0, 0x0, 0x0, 0xa0)
	/Users/markw/Code/go/src/github.com/versent/saml2aws/cmd/saml2aws/commands/login.go:220 +0x123
github.com/versent/saml2aws/cmd/saml2aws/commands.Login(0xc4201461c0, 0x5, 0x44d7ab7)
	/Users/markw/Code/go/src/github.com/versent/saml2aws/cmd/saml2aws/commands/login.go:103 +0x4cb
main.main()
	/Users/markw/Code/go/src/github.com/versent/saml2aws/cmd/saml2aws/main.go:73 +0x272
```

Version: saml2aws version 1.8.1
OS: macOS 10.12.5 (Sierra)
Provider: KeyCloak

**Comment by wolfeidau** [comment]
@dfevre yeah my bad, I will work on a fix now

**Comment by mksquared** [comment]
I have been working all morning to try to figure out if it was the app or not and try different ways to force it. Good find dfevre

**Comment by fogfish** [comment]
Exactly, same issue here from my side by different provider
```
saml2aws login --provider=JumpCloud
```

**Comment by wolfeidau** [comment]
This was fixed by #49 new release is out!

**Comment by fogfish** [comment]
thank you, it works now!
